app/models/category.rb

```diff
@@ -2,5 +2,5 @@ class Category < ApplicationRecord
   validates :name, presence: true, length: { minimum: 3, maximum: 15 }
   validates_uniqueness_of :name
   has_many :article_categories
-  has_many :articles, through: :article_categories
+  has_many :articles, through: :article_categories, dependent: :delete_all
```

Comment:
Und das löscht nur die Verknüpfung in articles_record, richtig? Und nicht die Artikel selber?
Und was würde passieren, wenn wir das dependent direkt an article_categories hängen würden? Nur so als Frage? Ich weiß es auch nicht.

Reply:
` Und was würde passieren, wenn wir das dependent direkt an article_categories hängen würden? `
Ich verstehe die Frage nicht. Meinst du article_categories model? Oder DB Tabelle?

Reply:
ach, du meinst Ziele 4!... also, weiß ich nicht, gute Frage!

Reply:
ja, genau, Zeile 4 :-)